### PR TITLE
Docs for *fromPath() methods.

### PR DESCRIPTION
**Pull request by dereuromark**
Docs for https://github.com/cakephp/cakephp/pull/14474

### CODE REVIEW

**Comment by dereuromark** [comment]
The docs also dont mention Router part, that should go into 4.x though.

File: en/appendices/4-1-migration-guide.rst [hunk 1]
```diff
@@ -164,6 +164,8 @@ View
 * Added ``TextHelper::slug()``. This method delegates to
   ``Cake\Utility\Text::slug()``.
 * Added ``ViewBuilder::addHelper()`` as chainable wrapper method to add helpers.
+* Added ``HtmlHelper::linkFromPath()`` and ``UrlHelper::urlFromPath()`` to build
+  links and URLs from route paths more easily and with IDE support in the View layer.
```
Comment (garas) [inline en/appendices/4-1-migration-guide.rst:168]: Docs for `Router::pathUrl()` / `*fromPath()` could include more details how to enable IDE support.

Comment (dereuromark) [reply en/appendices/4-1-migration-guide.rst:168]: Indeed, we could link from there then. The migration notes itself should keep it rather simple right?

Comment (garas) [reply en/appendices/4-1-migration-guide.rst:168]: The migration notes itself should be simple.

But as IDE support is not out of the box, but it is mentioned here, it needs explanation somewhere.

Comment (dereuromark) [reply en/appendices/4-1-migration-guide.rst:168]: You can put those into https://github.com/cakephp/docs/pull/6593 as that targets 4.x and can already be used for Router::pathUrl().
What comes here will just be merged into that.

**Pull request merged**